src/client/BaseApi.ts

```diff
@@ -189,6 +232,39 @@ export class BaseApi implements IAstarApi {
         }
     }
 
+    public async getStakerInfo(address: string): Promise<bigint> {
+        await this.ensureConnection();
+        let ss558Address = address;
+
+        if (isEthereumAddress(address)) {
+            const unifiedAccount = await this._api.query.unifiedAccounts.evmToNative<AccountId>(address);
```

Comment:
make sure that unifiedAccounts is deployed on a network, ie. it is not available on Astar and for sure it will come after staking v3.

`Object.prototype.hasOwnProperty.call(this._api.query, 'unifiedAccounts')`